packages/core/admin/admin/src/pages/Admin/Onboarding/tests/index.test.js

```diff
@@ -20,17 +20,24 @@ const App = (
 
 describe('Onboarding', () => {
   it('renders and matches the snapshot', async () => {
-    const {
-      container: { firstChild },
-    } = render(App);
+    const { getByRole, getByText } = render(App);
 
-    expect(firstChild).toMatchSnapshot();
+    fireEvent.click(getByRole('button', { name: /open help menu/i }));
+
+    expect(getByText(/get started videos/i)).toBeInTheDocument();
+    expect(document.body).toMatchSnapshot();
```

Comment:
Do you think we need the snapshot test? I think you are testing everything already more granulary :)